.deploy/helm/eupf/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: eupf
+version: 0.3.1
+appVersion: autoupdate
```

Comment:
Oh. Seems like it is a project's version. Right?

Reply:
app version (docker image tag)